internal/operator/operator.go

```diff
@@ -1178,6 +1182,8 @@ func (r *minioPolicyBindingReconciler) Reconcile(ctx context.Context, req reconc
 				User:     pb.Status.CurrentSpec.User.Builtin,
 			})
 		}
+		err = ignoreMadminErrorCode(err, "XMinioAdminNoSuchUser")
```

Comment:
the fix for `MinioGroupBinding` deletion is to catch and ignore errors when the user or group (for the policy attachment) no longer exists.